SU-SWS/decanter
# ADAPT-1771: Linting Configuration

# READY FOR REVIEW

# Summary
- Runs linters

# Needed By (Date)
- Whenever 
- 
# Urgency
- 0

# Steps to Test

1. Review code
2. Review build preview
3. Check out this branch on your local
4. Update your npm modules (nvm use && npm install)
5. Run the linter (npm run lint) and validate no linting errors.
